backend/controllers/channels.controller.js

```diff
@@ -0,0 +1,42 @@
+import Channel from '../models/channels.model.js'
+
+//@route   POST api/channels
+//@desc    Add New Channel
+//@access  Private
+
```

Comment:
can add a get request here too for the same.. Even if not using right now, can be helpful for a GUI admin panel.

Reply:
Yes, definitely those 2 api's would be useful in admin panel of this app.
So will make it in next version.
Thanks :)